src/firefly/js/charts/ChartUtil.js

```diff
@@ -548,6 +548,19 @@ export function applyDefaults(chartData={}) {
     chartData.layout = merge(defaultLayout, chartData.layout);
 }
 
+const TRACE_COLORS = ['lightblue', 'green', 'purple', 'brown', 'yellow', 'red'];
+export function getNewTraceDefaults(type='', traceNum=0) {
+    if (type.includes(SCATTER)) {
```

Comment:
I think, when the type is not SCATTER this function returns undefined, which is causing the type error, which I mentioned before (when we are trying to use the result as an array).